pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = [
+    "setuptools>=61.0.0",
```

Comment:
This breaks the installation in Python 2.7. Can you add a lower version specifically for that one?

Reply:
Done, @StefanRijnhart should I put `python_version < '2.8'`, or is it OK this way no matter the revision?

Reply:
Normally `setuptools` should be sufficient, since pip will select the latest version that is compatible with the python version used. Same for `setuptools_scm`.